### ISSUE TITLE
Problem when training custom model

### ISSUE THREAD

**Issue by jcole328** [AlexeyAB/darknet#5078]
Hello,

Not sure if my problem occurs because darknet.exe was built in the wrong file, but whenver I try to run the training cmd I get the error "Couldn't open file: yolo-obj.cfg. I tried copying this file to the outer darknet file's cfg folder instead of the one in the x64 build but that still yields the same problem. Here is a picture of the last three times I tried running this command:
![image](https://user-images.githubusercontent.com/38474853/77213493-3a292580-6ac8-11ea-822c-442f4528a875.png)

Side note, I am trying to get this to run on my GPU. My CUDA and cuDNN files are up to date so I'm not sure why it isn't using my GPU.

Any help would be greatly appreciated!
Thanks!

**Comment by killypham** [comment]
1) you should run:
   ./darknet detector train ....... 
or darknet.exe detector train .....
2) Please check the location of the cfg file. Maybe you made a mistake.

**Comment by jcole328** [comment]
No, yolo-obj.cfg is currently in both of the cfg folders because it wouldn't originally in the outside cfg folder. Being on windows I originally did darknet.exe but PowerShell told me to use .\darknet.exe for it to actually run.
